packages/aws-cdk-lib/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -262,6 +270,22 @@ export class Trail extends Resource {
       },
     }));
 
+    if (props.isOrganizationTrail) {
+      this.s3bucket.addToResourcePolicy(new iam.PolicyStatement({
+        resources: [this.s3bucket.arnForObjects(
+          `AWSLogs/${props.orgId}/*`,
```

Comment:
Looking at the below snippet, wouldn't the `orgId` be `this.s3bucket.stack.account`?
```
Resource": "arn:aws:s3:::myBucketName/[optionalPrefix]/AWSLogs/myAccountID/*",
            "Condition": {
                "StringEquals": {
                    "s3:x-amz-acl": "bucket-owner-full-control",
                    "aws:SourceArn": "arn:aws:cloudtrail:region:myAccountID:trail/trailName"
                }
            }
```

Reply:
That's a different permission from that one we are adding here:
```
"Resource": "arn:aws:s3:::myOrganizationBucket/AWSLogs/o-organizationID/*",
            "Condition": {
                "StringEquals": {
                    "s3:x-amz-acl": "bucket-owner-full-control",
                    "aws:SourceArn": "arn:aws:cloudtrail:region:managementAccountID:trail/trailName"
                }
            }
```